### PR TITLE
Added "RESOURCE EXCHAUSED" exception name

### PR DESCRIPTION
**Pull request by berndruecker**
This allows people to relate their problem to this concept better

### CODE REVIEW

File: docs/components/zeebe/technical-concepts/internal-processing.md [hunk 1]
```diff
@@ -57,16 +57,15 @@ This command can in turn be processed, completing the service task and driving t
...
+The maximum rate of requests that can be processed by a broker depends on the processing capacity of the machine, the network latency, current load of the system, etc. There is no fixed limit configured in Zeebe for the maximum rate of requests it accepts. Instead, Zeebe uses an adaptive algorithm to dynamically determine the limit of the number of in-flight requests (the requests that are accepted by the broker, but not yet processed).
 
-The inflight request count is incremented when a request is accepted, and decremented when a response is sent back to the client. The broker rejects requests when the inflight request count reaches the limit.
+The in-flight request count is incremented when a request is accepted, and decremented when a response is sent back to the client. The broker rejects requests when the in-flight request count reaches the limit.
 
-When the broker rejects requests due to backpressure, the clients can retry them with an appropriate retry strategy. If the rejection rate is high, it indicates that the broker is constantly under high load.
-In that case, it is recommended to reduce the request rate.
+In that case, it is recommended to .
```
Comment (christinaausley) [inline docs/components/zeebe/technical-concepts/internal-processing.md:71]: I think we might have cut this sentence short.

File: docs/components/zeebe/technical-concepts/internal-processing.md [hunk 2]
```diff
@@ -57,16 +57,15 @@ This command can in turn be processed, completing the service task and driving t
...
 
 To avoid such problems, Zeebe employs a [backpressure](/self-managed/zeebe-deployment/operations/backpressure.md) mechanism.
 When the broker receives more requests than it can process with an acceptable latency, it rejects some requests.
 
-The maximum rate of requests that can be processed by a broker depends on the processing capacity of the machine, the network latency, current load of the system, etc.
+This is indicated to the client by a **resource exhausted** exception. If a client sees this exception, it can retry the requests with an appropriate retry strategy. If the rejection rate is high, it indicates the broker is constantly under high load and you need to reduce the rate of requests. Alternatively, you can also increase broker resources to adjust to your needs. In high-load scenarios, it is recommended to [benchmark](https://camunda.com/blog/2022/05/how-to-benchmark-your-camunda-platform-8-cluster/) your Zeebe broker up front to size it correctly.
```
Comment (christinaausley) [inline docs/components/zeebe/technical-concepts/internal-processing.md:65]: What is "This" in "this is indicated by..."? Do you mean "These rejected requests are indicated to the client by a..." or "This backpressure mechanism is indicated to the client by a..."?

**Review by christinaausley — CHANGES_REQUESTED** [review]
Very minor adjustments in my latest commit and two comments for clarification. Once these are resolved LGTM.

File: docs/components/zeebe/technical-concepts/internal-processing.md (hunk 2 above)
Comment (berndruecker) [reply docs/components/zeebe/technical-concepts/internal-processing.md:65]: "Backpressure is indicated"

File: docs/components/zeebe/technical-concepts/internal-processing.md (hunk 1 above)
Comment (berndruecker) [reply docs/components/zeebe/technical-concepts/internal-processing.md:71]: Hups - I think I moved the content and wanted to remove the sentence. I will fix this

**Review by christinaausley — APPROVED** [review]
Thanks @berndruecker ! LGTM.

**Comment by akeller** [comment]
@christinaausley since @berndruecker is out, do you want to go ahead and merge this? Any idea if he had more to work on here?

**Comment by christinaausley** [comment]
> @christinaausley since @berndruecker is out, do you want to go ahead and merge this? Any idea if he had more to work on here?

I don't believe he had anything additional to add. If so, we can always iterate on a new branch given this one is growing quite old. Will merge.

**Pull request merged**